tests/test_misc.lua

```diff
@@ -541,4 +542,382 @@ T['zoom()']['respects `config` argument'] = function()
   child.expect_screenshot()
 end
 
+T['setup_restore_cursor()'] = new_set()
+
+local setup_restore_cursor = function(...) child.lua('MiniMisc.setup_restore_cursor(...)', { ... }) end
+
+local restore_cursor_default_opts = {
```

Comment:
Most usage of `restore_cursor_default_opts` seems to be to indicate that `restore_cursor()` is called with default options. Cleaner approach is to call it without arguments at all.

Reply:
Fixed.